lab-peter/test/fs-readfile-test.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const readFile = require('../lib/readfile');
+const assert = require('assert');
+
+describe('testing index', function() {
+  describe('#readFile()', function() {
+    it ('should read one.txt two.txt three.txt', function(done) {
+      readFile(function(myArray) {
+        assert.equal(myArray.join(),['52616e646f6d2074','536f6d6520616464','46696e616c6c7920']); //text is your myArray
```

Comment:
I don't think that this should be passing. myArray.join() will produce a string (of all of the items in the array concatenated), and a string is not equal to an array. Are you sure that you're not getting a false positive? Please check this out!